tst/CTA.Rules.Test/Models/ExpectedOutputConstants.cs

```diff
@@ -834,5 +834,817 @@ public void OwinDataHandler(ITextEncoder encoder, SecureDataFormat<Authenticatio
         }
     }
 }";
+        public const string WCFConfigStartup =
+@"
```

Comment:
I create an issue https://github.com/aws/cta/issues/313 ... we should do that longterm